.travis.yml

```diff
@@ -2,17 +2,22 @@ sudo: false
 language: node_js
 cache:
   directories:
-    - node_modules
+  - node_modules
```

Comment:
mmm, travis CLI did that when I added the encrypted tokens through it `travis encrypt --add` so I thought it should be ok...